File: c++/Program-1.cpp
//Program-1 :- Write a program to check whether a number is od or even using bitwise operator.

#include<iostream>
using namespace std;


int main(){
 int num;


     cout<<"Enter Number \n";
     cin>>num;
     if((num &1) == 0){
         cout<<num<<" is even \n";
     }else{
         cout<<num<<" is odd \n";
     }

 
}

File: java/Program_1.java
import java.util.Scanner;

//Program-1 :- Write a program to check whether a number is odd or even using bitwise operator.

class Program_1{
    public static void main(String arg[]){
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter a number");
        int n=sc.nextInt();
        if((n & 1)==0){
            System.out.println(n + " is even");
        }else{
            System.out.println(n + " is odd");
        }
    }
}